pipelines/internal/commands/query/query.go

```diff
@@ -40,8 +41,7 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 	call := service.Projects.Operations.List(path).Context(ctx)
 
 	if !*all {
-		s := []string{*filter,"done = false"}
-		*filter = strings.Join(s, " ")
+		*filter = strings.Join([]string{*filter,"done=false"}, " ")
```

Comment:
Make sure you run both gofmt and goimports.  There are two errors that those tools will correct (whitespace here and the extra line in the imports section)